application/src/main/java/org/opentripplanner/transit/service/DefaultTransitService.java

```diff
@@ -183,9 +186,8 @@ public RegularStop getRegularStop(FeedScopedId id) {
 
   @Override
   public Route getRoute(FeedScopedId id) {
-    TimetableSnapshot currentSnapshot = lazyGetTimeTableSnapShot();
-    if (currentSnapshot != null) {
-      Route realtimeAddedRoute = currentSnapshot.getRealtimeAddedRoute(id);
+    if (this.timetableSnapshot != null) {
```

Comment:
Not very important, but why do you prefix the field with "this"?

Reply:
I didn't do anything - it was all IntelliJ! :)

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/6363/commits/c740b8302d914d2a6c7bb0f59284018ddfad6631